deps/rabbitmq_auth_backend_oauth2/ebin/jwks_http.app

```diff
@@ -0,0 +1,9 @@
+{application, 'jwks_http', [
```

Comment:
Seems like these are generally generated by erlang.mk, and `make clean` tramples all over this... I assume there's a better way to do this?

Reply:
I'd try starting a Cowboy server with some routes right on the Common Test runner node. We just need a local HTTP endpoint, right? It doesn't really have to be a separate Erlang node?

Reply:
Yep, that's correct. I mostly did that because I was following the `trust_store` environment, which seemed to tie cowboy's lifetime into the `app`'s, I think? Guess we need to shut down Cowboy some other way otherwise.

Reply:
543e8aa does away with `jwks_http.app`.